src/index.html

```diff
@@ -1,14 +1,19 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Colored blocks</title>
   <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-  <h1>Colored blocks</h1>
+  <div class="color1 general"></div>
+  <div class="color2 general"></div>
+  <div class="color3 general"></div>
```

Comment:
what is color1, color2 and so on? numbers don't show difference between elements

those should be `outer-box`, `middle-box`, `inner-box` for example